relay-server/src/endpoints/healthcheck.rs

```diff
@@ -32,11 +33,15 @@ impl HealthcheckResponse {
 }
 
 fn healthcheck_impl(message: IsHealthy) -> ResponseFuture<HttpResponse, Error> {
+    let fut = async move {
+        let addr = Healthcheck::from_registry();
+        addr.send(message.into()).await
+    };
+    let bp = Box::pin(fut);
```

Comment:
There's a [`boxed_local`](https://docs.rs/futures/latest/futures/future/trait.FutureExt.html#method.boxed_local) function that achieves this for you. You could call it like this:

```rust
bp
    .boxed_local()
    .compat()
    .map_err() ...
```

Aside, do we need the `async move` block here because of the `&self` on `Addr::send`?